media/js/mozorg/home/home.js

```diff
@@ -66,4 +70,68 @@
 
     // Video card interactions.
     initVideoEvents();
-})();
+
+    /*
+     * Sticky CTA
+     */
+
+
+    var $stickyCTA = $(document.getElementById('download-firefox-sticky-cta'));
```

Comment:
Nit pick - could just do `$('#download-firefox-sticky-cta');`

Reply:
I believe this is faster but I can't find a citation for that.

Reply:
I know `getElementById` is faster than `querySelector`, and both are faster than jQuery's `$` lookup, but I don't know about the `$` `+` `getElementById` combo.